### ISSUE TITLE
Create admin account for clients + trial plan

### ISSUE THREAD

**Issue by ana-0k** [ubiquity/partnerships#86]
Even though we have a select target client type, the current [Pilot Program Google Form](https://docs.google.com/forms/d/e/1FAIpQLSedf-TKN3hRbc18HlmgY2j8VAb_4KYn_P5z6h0E1MaH1ieq0A/viewform) is very poor UX, I'm sure it will prevent targets from signing up.

We need to make client UX as seamless and close to automatic experience as possible.
i.e. People think they sign up for the system as via common website UI, input all required info, including legal, and the "verification" process is triggered. In the background, we manually check what kind of organization it is and how soon/what service are we willing to suggest.
  *when things even **seem** automatic, people are more likely to sign up (verified by me)

Based on the above, there may be 2 types of accounts:
- **Full use of the system** (for your wishlist clients, but later when it's fully ready and scaled)
- **Partial use - trial account** (described below, flow idea from experience with ad monetization platforms)

-----------------------
**Trial Account Summary**

- A traditional button on the main website to sign up/login
- Comment on top of the page that only trial accounts with limited task number are currently available
- Input name, email, verify email with the code
- After simple sign-up, user is instantly led to the dashboard/account
- Maybe a small 4-step on-screen tutorial ("here you can create tasks", "here you can invite contributors", "here you can track progress", "now you need to verify your account" > Go)
- Type in company details (legal name, website, company size, industry, HQ country)
- Submit legal documents (biz license)
- 'Verification in progress' status starts

**_Can be just until this step before the system seems to be operational enough for next steps_**

- We check the submitted info internally, see if we want to approve them or not
- If approved, can open like up to 5 new task slots a month 
   *limiting number of new tasks will reduce requirements for customer support
   *this way, we accumulate small traction with different kinds of businesses and test/improve the system with minimal resources
   *good to test the system before committing to large organizations like SKY/etc cuz they are gonna want to see traction/proof of efficiency for potential partner project that were not supported/developed internally
- Extra task slots can be opened if the partner aligns with target
- Before/upon creating the first task, user is requested to top up payout balance
   *simply connecting a wallet is not secure cuz it may not have sufficient funds at the time of payment to their contributors
- Small on-screen step-by-step guide first time the task is created
   *set importance, time, deadline, is it a one-time or repeated task, see if the automatically suggested price is correct, add hashtag, topic & text
- Suggest checking best practices for writing efficient task scripts in our manual 
- Suggest managing general pricing somewhere in the menu
- Suggest adding plugins

**_Gonna need to add a lot of UI on top of GitHub, probably. Or simply do the trials on GitHub until UI is ready?_**

**Comment by 0x4007** [comment]
> Or simply do the trials on GitHub until UI is ready?

Yes we definitely prefer the lean approach. I do not want to make any extra unnecessary UI if we can pull it off on GitHub directly. We could even make a new repo where applicants file an issue to apply. There are issue templates support, which can allow us to cater the UX a bit more see [here](https://github.com/ubiquity/ubiquibar/issues/new/choose). 

> - Type in company details (legal name, website, company size, industry, HQ country)

I wonder if this is even necessary for now. The industry is quite small. We would probably know these details roughly just by their org name

> - Submit legal documents (biz license)

We have plans for KYC verification, perhaps we should find a KYB/KYC provider. https://github.com/ubiquity-os/plugins-wishlist/issues/24

Somewhat related
- https://github.com/ubiquity-os/ubiquity-os-kernel/issues/90#issuecomment-2292995655
